mvrx-mocking/src/main/kotlin/com/airbnb/mvrx/mocking/MockBuilder.kt

```diff
@@ -1436,14 +1440,21 @@ open class MavericksViewMocks<V : MockableMavericksView, Args : Parcelable> @Pub
 
         private var allowCreationForTesting: Boolean = false
 
+        /**
+         * A Provider of [MavericksViewMocks]
+         * When set [mockProvider] will be used to retrieve mocks first before calling [MockableMavericksView.provideMocks]
+         */
+        var mockProvider:ViewMocksProvider? = null
```

Comment:
it might make sense to have this not be nullable and instead default to an implementation that does `view.call("provideMocks")`

Reply:
Made a stab at it, its a bit tough to do cleaner as the DefaultViewMocksProvider still needs to be delegated to if the override does not return a mock. Let me know what you think of the change, I'm not married to either.